authn_k8s_scale_demo/dev_deploy.sh

```diff
@@ -2,20 +2,25 @@
 
 source ./evokecmd.sh
 
+kubectl config use-context conjur
+
 echo Grabbing the conjur.pem
 ssl_certificate=$(evokecmd cat /opt/conjur/etc/ssl/ca.pem)
 
+kubectl config use-context minikube
+
 echo Storing non-secret configuration data
 
-# write out host factory name & token, variable name and deployment name in configmap
-kubectl delete configmap webapp
+# write out conjur ssl cert in configmap
+kubectl delete --ignore-not-found=true configmap webapp
```

Comment:
Are these deletions necessary to make script runs idempotent? Or to prevent having to start from scratch when the script crashes halfway through perhaps?

Reply:
There might be a cleaner way to update it, but delete and re-create is just an approach that worked for me.